### DISCUSSION TITLE
How to manually select the platform rendering method (mac, visionos, ipados, etc)?

### DISCUSSION THREAD

**Discussion by fredrikburmester** [callstack/react-native-bottom-tabs#169] [Q&A] [answered]
I'd like to have a TV or iPad looking tab bar but on Mac, is that possible? Currently the tab bar on mac is a sidebar and i'd really like the floating buttons.

**Comment by okwasniewski** [comment, marked as answer]
Hey, unfortunately it's not possible as it due to the internal Apple system implementation to determine the look per platform

**Comment by fredrikburmester** [comment]
Is it possible to set the default behavior for iPad? Right now it defaults to the side but i always want it floating on top.

**Reply by okwasniewski** [reply]
Hey, `sidebarAdaptable` should be responsible for this. Also, the floating on top behaviour is available from iPadOS 18+ (before that it's on the bottom)
